container/sagemaker/serve.py

```diff
@@ -125,6 +157,15 @@ def _start_tfs(self):
         log.info('started tensorflow serving (pid: %d)', p.pid)
         self._tfs = p
 
+    def _start_python_service(self):
+        self._log_version('gunicorn --version', 'gunicorn version info:')
+        cmd = 'gunicorn -b unix:/tmp/gunicorn.sock --chdir /sagemaker ' \
+              '--pythonpath {} python_service:app --reload'.format(self._python_lib_dir)
```

Comment:
don't need --reload... code shouldn't change after service is launched, and this option creates a bit of overhead